imap-proto/src/parser/core.rs

```diff
@@ -1,4 +1,9 @@
-use nom::{character::streaming::digit1, IResult};
+use nom::{
+    bytes::streaming::{tag, take},
```

Comment:
This change doesn't make sense anymore because there are now a bunch of things being imported that don't get used.